packages/strapi-generate-new/files/config/.init.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We could just create a file called `.init` without an extension like a `.htaccess` without any content into it?

Reply:
I don't know how `glob` will react but it makes more sense to me. What do you think?